http/client.go

```diff
@@ -0,0 +1,68 @@
+package http
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
```

Comment:
It would be nice to have some default arguments that we could use in our integration as flags/environment variables. Ex:
```go
package main

...

type argumentList struct {
    sdkArgs.DefaultArgumentList

    http.ClientArguments
}
```
and they would expose: 
`CA_BUNDLE_FILE`, `CA_BUNDLE_DIR` and `HTTP_TIMEOUT` vars.

I think that could avoid some toil when we're developing integrations.

Reply:
http.ClientArguments would be something like:

```go
type ClientArguments struct {
    CA_BUNDLE_FILE string `default: "" help: "blabla"`
    CA_BUNDLE_DIR string `default: "" help: "blabla"`
    HTTP_TIMEOUT int `default:30 help:"blabla"`
}
```

Reply:
👍  But i would add `HTTP` prefix to the struct name

Reply:
@areina I'm not agree on using `sdkArgs.DefaultArgumentList` this should not depend on the SDK cli args, but just work as a helper within its bounded domain (https requests).